react/components/molecules/chat-message/chat-message.js

```diff
@@ -57,9 +61,18 @@ export class ChatMessage extends PureComponent {
         return { marginTop: index > 0 ? 2 : 0 };
     };
 
+    _chatMessageComponent = () => {
+        return this.props.onPress ? Touchable : View;
+    };
+
     render() {
+        const ChatMessageComponent = this._chatMessageComponent();
         return (
-            <View style={[styles.chatMessage, this.props.style]}>
+            <ChatMessageComponent
+                style={[styles.chatMessage, this.props.style]}
+                underlayColor={this.props.underlayColor}

```

Comment:
This will require changing more stuff 😕 

https://user-images.githubusercontent.com/25725586/161313513-10d8231c-256f-4bb7-8e2e-058b94b9509c.mov